vanilla_installer/gui.py

```diff
@@ -27,7 +27,8 @@
 )
 
```

Comment:
this import got reverted by the main branch merge in the previous commit
```suggestion
import platform
```